vscode-dotnet-runtime-library/src/Acquisition/DotnetConditionValidator.ts

```diff
@@ -136,22 +136,38 @@ Please set the PATH to a dotnet host that matches the architecture ${requirement
         return os.platform() === 'win32' ? (await this.executor!.tryFindWorkingCommand([CommandExecutor.makeCommand('chcp', ['65001'])])) !== null : false;
     }
 
-    private stringVersionMeetsRequirement(foundVersion : string, requiredVersion : string, requirement : DotnetVersionSpecRequirement) : boolean
+    private stringVersionMeetsRequirement(availableVersion : string, requestedVersion : string, requirement : DotnetVersionSpecRequirement) : boolean
     {

```

Comment:
can you use semver for comparison?

Reply:
That's a good suggestion, `semver` is a good option. I hadn't considered that, thank you. I wish we were using semver in the code a lot earlier in many places. 

Going forward I will use that. I do want to get this merged quickly though, so I'm not sure about making that change now considering this is already written and tested 🤔

Reply:
Can you file an issue to use semver rather than parsing versions ourselves?

Reply:
Yes, good idea :eyes: https://github.com/dotnet/vscode-dotnet-runtime/issues/2003